Sources/SwiftIDEUtils/DeclNameLocation.swift

```diff
@@ -102,6 +102,12 @@ public struct DeclNameLocation: Equatable {
     /// The parameter of a function declaration, like `func foo(a b: Int)`
     case parameters([Argument])
 
+    /// An enum ase declaration like `case myCase(label: String)`.
```

Comment:
```suggestion
    /// An enum case declaration like `case myCase(label: String)`.
```

Reply:
I swear I made this type 10 times while writing the PR. One of them seems to have slipped through 😆